src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -256,6 +338,14 @@
     crafted: true
 """
 
+helps['backup recoverypoint show-log-chain'] = """
+type: command
+short-summary: This command lists the start and end points of the unbroken log chain(s) of the given backup item.
```

Comment:
The same.

Reply:
@sambitratha : "List the start and end points of the unbroken log chain(s) of the given Azure workload backup item"